pages/linux/add-apt-repository.md

```diff
@@ -0,0 +1,19 @@
+# add-apt-repository
+
+> Manages apt repository definitions.
+
+- Add a new apt repository:
+
+`add-add-repository {{repository_spec}}`
```

Comment:
`add-add-repository` => `add-apt-repository` 😉

Reply:
Thanks - good spot! I think I got myself a bit confused, because you can use `add-apt-repository` and `apt-add-repository` interchangeably. They even have the same man page O.o